dashboard/test/controllers/code_reviews_controller_test.rb

```diff
@@ -0,0 +1,113 @@
+require 'test_helper'
+
+class CodeReviewsControllerTest < ActionController::TestCase
+  self.use_transactional_test_case = true
+
+  setup_all do
+    @project_owner = create :student
+    @storage_id = create_storage_id_for_user(@project_owner.id)
+    @channel_id = create :project, storage_id: @storage_id
+    _,  @project_id = storage_decrypt_channel_id(@channel_id)
+  end
+
+  setup do
+    sign_in @project_owner
+  end
+
+  test 'index returns both open and closed code reviews' do
```

Comment:
Options: But not deleted reviews?

Reply:
Just realized that was not a full sentence that I left! I meant optionally, it could be worth also testing that deleted code reviews are not returned in the response. This is definitely not blocking though

Reply:
Got it, good suggestion!  I'll pick it up in the next PR.